model/Core/Vocabularies/RelationshipType.md

```diff
@@ -44,16 +44,16 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - example: Every `to` Element is an example for the `from` Element
 - evidenceFor: (Dataset) Every `to` Element is can be considered as evidence for the `from` Element
 - expandedFromArchive: Every `to` Element is an artifact expanded from the `from` archive file
-- exploitCreatedBy: (Security) Designates an agent has created an exploit against a vulnerability
+- exploitCreatedBy: (Security) The `to` Agent(s) has created an exploit against a `from` Vulnerability element
 - fileAdded: Every `to` Element is is a file added to the `from` Element
 - fileDeleted: Every `to` Element is a file deleted from the `from` Element
 - fileModified: Every `to` Element is a modification of the `from` Element
-- fixedBy: (Security) Designates a vulnerability has been fixed by an agent
-- fixedIn: (Security/VEX) A vulnerability has been fixed in one or more elements
-- foundBy: (Security) Designates an agent was the original discoverer of a security vulnerability
+- fixedBy: (Security) Designates a `from` Vulnerability element has been fixed by the `to` Agent(s)
+- fixedIn: (Security/VEX) A `from` Vulnerability element has been fixed in each of the `to` Element(s)
+- foundBy: (Security) Designates a `from` Vulnerability was originally discovered by the `to` Agent(s)
 - generates: Every `to` Element is generated from the `from` Element
-- hasAssessmentFor: (Security) Relates a Vulnerability and an Element with a security assessment.
-- hasAssociatedVulnerability: (Security) Used to associate a security vulnerability with a software artifact
```

Comment:
For alignment with the model - we should probably say its "with an Artifact" rather than "with a software artifact".